### ISSUE TITLE
Fetching properties from nested arrays returns undefined

### ISSUE THREAD

**Issue by egmacke** [justinlettau/ts-dot-prop#32]
## Describe the Issue

When trying to get the property from objects nested within multiple levels of arrays, results in undefined.

## Expected behavior

Should return nested arrays reflecting the defined path.

## Steps to Reproduce

```
const object = { 
  arr: [
    {
      content: [
        {a: 1, b: 2, c: 3},
        {a: 4, b: 5, c: 6}
      ]
    },
    {
      content: [
        {a: 6, b: 7, c: 8},
        {a: 9, b: 10, c:11}
      ]
    }
  ]
}
get(object, 'arr[*].content[*].a') // Expect: [ [ 1, 4 ], [ 6, 9 ] ], Actual: [ undefined, undefined ] 
```

**Comment by egmacke** [comment]
@justinlettau are you happy to accept this change? And if so could we get a new release that includes it so we can use a released version in our code.

Happy to help make this happen if that would help?

**Comment by justinlettau** [comment]
@egmacke Accepted and released in v2.1.0! Thank you! 🎉
